### ISSUE TITLE
serviceException element contains invalid variables element.

### ISSUE THREAD

**Issue by jmacauley** [NORDUnet/opennsa#4]
OpenNSA will return a serviceException element with a variable formatted as follows:

    <?xml version="1.0" encoding="UTF-8" standalone="no"?>
    <soap:Envelope xmlns:ftypes="http://schemas.ogf.org/nsi/2013/12/framework/types" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
        <soap:Header />
        <soap:Body>
            <soap:Fault>
                <faultcode>soap:Server</faultcode>
                <faultstring>Invalid request: Start time in the past (Startime: 2016-11-03 14:15:00, Delta: 5:49:52.841370)</faultstring>
                <detail>
                    <ftypes:serviceException>
                        <nsaId>urn:ogf:network:icair.org:2013:nsa</nsaId>
                        <errorId>00100</errorId>
                        <text>Invalid request: Start time in the past (Startime: 2016-11-03 14:15:00, Delta: 5:49:52.841370)</text>
                        <variables>
                            <variables type="startTime">
                                <value>2016-11-03T14:15:00</value>
                            </variables>
                        </variables>
                    </ftypes:serviceException>
                </detail>
            </soap:Fault>
        </soap:Body>
    </soap:Envelope>

The `<variables>` element should contain a `<variable>` element and not another `<variables>` element as follows:

                        <variables>
                            <variable type="startTime">
                                <value>2016-11-03T14:15:00</value>
                            </variable>
                        </variables>

The issue is locaed in: `opennsa/protocols/nsi2/bindings/nsiframework.py`
```
@@ -175,7 +175,7 @@ class ServiceExceptionType(object):
        ET.SubElement(r, 'errorId').text = self.errorId
        ET.SubElement(r, 'text').text = self.text
        if self.variables is not None:
-          ET.SubElement(r, 'variables').extend( [ e.xml('variables') for e in self.variables ] )
+          ET.SubElement(r, 'variables').extend( [ e.xml('variable') for e in self.variables ] )
        if self.childException is not None:
            for el in self.childException:
                ET.SubElement(r, 'childException').extend( el.xml('childException') )
```

**Comment by htj** [comment]
Thanks for reporting + patch.

Should be fixed in commit 1ed3758f1c39ced0cd6e0181af171f0c04e21c6c

**Comment by jmacauley** [comment]
No problem.  Thank you for fixing so quickly.  I attempted a pull request to make it even easier but didn't have permissions.  Will notify AGOLE participants to upgrade at their leisure after SC16.

Thanks again.
